app/controllers/api/v2/job_invocations_controller.rb

```diff
@@ -201,7 +201,7 @@ def job_invocation_params
       def composer_for_feature
         JobInvocationComposer.for_feature(
           job_invocation_params[:feature],
-          job_invocation_params[:host_ids],
+          job_invocation_params[:search_query],
```

Comment:
No problem at all. Should I add host_ids as a param to the controller method with the description stating that it's deprecated ?

Reply:
If I remember correctly the original purpose of it was to allow katello's angular ui to trigger rex jobs for package actions. The param wasn't exposed because if was considered internal. I can't really say that I like the idea of documenting a param which shouldn't be used by the wider public just to deprecate it

Reply:
Ok. Katello doesn't use the REST API but rather we call JobInvocationComposer directly. We do send host ids, though.

Reply:
I may be misremembering or it might be something that was there some years ago but was eventually removed.